kuma/static/styles/includes-skinny/_mixins.scss

```diff
@@ -377,19 +377,17 @@ These are not dynamic but serve as mixins
 
 @mixin heading-1() {
     @include set-font-size($h1-font-size);
-    font-family: $heading-font-family;
+    @include set-heading-font-family();
     line-height: 1;
-    #{$selector-heading-font-fallback} {
-        font-family: $heading-font-family-fallback;
-    }
 }
 
 @mixin heading-2() {
-    @include set-font-size($h2-font-size);
-    font-family: $heading-font-family;
-    line-height: 1;
-    #{$selector-heading-font-fallback} {
-        font-family: $heading-font-family-fallback;
+    @include set-font-size($mobile-h2-font-size);
```

Comment:
Tests are failing because `$mobile-h2-font-size: 32px;` is missing from `kuma/static/styles/includes-skinny/_vars.scss`